server/status_test.go

```diff
@@ -46,18 +46,30 @@ func startStatusServer() *httptest.Server {
 	return httpServer
 }
 
-// TestStatusLocalStacks verifies that goroutine stack traces are available
-// via the /_status/local/stacks endpoint.
-func TestStatusLocalStacks(t *testing.T) {
+// TestStatus verifies that status endpoints return expected results.
+func TestStatus(t *testing.T) {
```

Comment:
It would be good to test different encodings as well, since that is the new functionality you've added.